spynl/main/mail.py

```diff
@@ -146,36 +146,34 @@ def send_template_email(request, recipient, template_string=None,
     if subject:
         replacements['subject'] = subject
 
-    for key, value in replacements.items():
-        if isinstance(value, SpynlTranslationString):
-            replacements[key] = value.translate()
-
     if template_file is not None:
         try:
-            replacements['content'] = render(template_file, replacements,
-                                             request=request)
+            html_body = render(template_file + '.jinja2', replacements,
+                               request=request)
+            if not subject:
+                subject = render(template_file + '.subject.jinja2',
+                                 replacements, request=request)
+                subject = subject.replace('\n', '')
         except TemplateNotFound:
+            # todo: change message in error:
             raise EmailTemplateNotFound(template_file)
```

Comment:
should we specify which template was not fount (template or subject), or should we just add an empty subject if there is no subject template?

Reply:
This is spynl.main and so it should not have Softwear connect in there.